### PR TITLE
feat: proof request list

### PR DESCRIPTION
**Pull request by nikita-zakharov-dsr**
# Summary of Changes
Add proof requests list, update navigation, add Proof Request Full Name screen (placeholder)
image
image

# Related Issues

https://jira.dsr-corporation.com/browse/BCGOVSSIMO-3

# Pull Request Checklist

Tick all boxes below to demonstrate that you have completed the respective task. If the item does not apply to your this PR **check it anyway** to make it apparent that there's nothing to do.

- [ ] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this);
- [ ] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components;
- [ ] Updated documentation as needed for changed code and new or modified features;
- [ ] Added sufficient [tests](../__tests__/) so that overall code coverage is not reduced.

If you have _any_ questions to _any_ of the points above, just **submit and ask**! This checklist is here to _help_ you, not to deter you from contributing!

Pro Tip 🤓

- Read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!
- Your PR will likely not be reviewed until all the above boxes are checked and all automated tests have passed.

_PR template adapted from the Python attrs project._

### CODE REVIEW

File: core/App/localization/en/index.ts [hunk 1]
```diff
@@ -347,6 +347,7 @@ const translation = {
     "ConfirmDecline": "Yes, decline this proof request",
     "AbortDecline": "No, go back",
     "DeclineTitle": "Decline Proof",
+    "ZeroKnowledgeProof": "Zero-knowledge proof"
```
Comment (Artemkaaas) [inline core/App/localization/en/index.ts:350]: Please move new labels under `Verifier` object

File: core/App/navigators/ProofRequestStack.tsx [hunk 2]
```diff
@@ -17,6 +18,13 @@ const ProofRequestStack: React.FC = () => {
   const defaultStackOptions = createDefaultStackOptions(theme)
   return (
     <Stack.Navigator screenOptions={{ ...defaultStackOptions }}>
+      <Stack.Screen
+        name={Screens.ProofRequestFullName}
```
Comment (Artemkaaas) [inline core/App/navigators/ProofRequestStack.tsx:22]: Let's call this screen and component `ProofRequestDetails`

File: core/App/screens/ListProofRequests.tsx [hunk 3]
```diff
@@ -1,16 +1,87 @@
...
+
+const iconSize = 35
+const iconName = 'chevron-right'
+
 const ListProofRequests: React.FC<ListProofRequestsProps> = ({ navigation }) => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const { t } = useTranslation()
   const records = defaultProofRequestTemplates
-  return <></>
+  const hasPredicates = (record: ProofRequestTemplate): boolean => {
```
Comment (Artemkaaas) [inline core/App/screens/ListProofRequests.tsx:58]: move this function to `utils` proof-request file

**Review by Artemkaaas — CHANGES_REQUESTED** [review]

File: core/App/localization/en/index.ts (hunk 1 above)
Comment (nikita-zakharov-dsr) [reply core/App/localization/en/index.ts]: done

File: core/App/navigators/ProofRequestStack.tsx (hunk 2 above)
Comment (nikita-zakharov-dsr) [reply core/App/navigators/ProofRequestStack.tsx]: done

**Review by Artemkaaas — APPROVED** [review]

**Pull request merged**